scripts/ci/vagrant.sh

```diff
@@ -19,7 +19,7 @@ setup() {
 	# Tar up the git checkout to have vagrant rsync it to the VM
 	tar cf criu.tar ../../../criu
 	# Cirrus has problems with the following certificate.
-	wget --no-check-certificate https://releases.hashicorp.com/vagrant/${VAGRANT_VERSION}/vagrant_${VAGRANT_VERSION}_"$(uname -m)".deb -O /tmp/vagrant.deb && \
+	wget --no-check-certificate https://releases.hashicorp.com/vagrant/${VAGRANT_VERSION}/vagrant_${VAGRANT_VERSION}-1_"$(dpkg --print-architecture)".deb -O /tmp/vagrant.deb && \
```

Comment:
What is the reason for this change?

Reply:
It looks like they have changed the format of the URL:
https://releases.hashicorp.com/vagrant/2.3.6/